src/action_management/include/actionmanagement.h

```diff
@@ -193,14 +193,15 @@ int action_direction(game_t *g, action_t *a, direction_t *d);
 /* A function that executes KIND 3 actions (ACTION <npc>)
  * 
  * Parameters:
+ * - g: A game struct
  * - a: An action struct
  * - n: An npc struct
  * 
  * Returns
  * - 0 on success
  * - 1 on failure
  */
-int action_npc(action_t *a, npc_t *n);
+int action_npc(game_t *g, action_t *a, npc_t *n);
```

Comment:
I don't think this should take in a game_t struct because it doesn't affect the game state at all. It only spits out whatever dialogue an npc has, and can do that multiple times, and doesn't change anything else in the game, so the game_t struct wouldn't be used within the function at all. 
```suggestion
int action_npc(action_t *a, npc_t *n);
```